### PR TITLE
Getting defaults config up to date

### PR DESCRIPTION
**Pull request by 51N84D**
Let's modify defaults.yaml here real quick

### CODE REVIEW

File: shared/trainer/defaults.yaml [hunk 1]
```diff
@@ -32,14 +32,14 @@ data:
...
       ignore: false
-      new_size: 256
+      new_size: 640
     - name: crop
-      ignore: false
-      height: 224
-      width: 224
+      ignore: true
+      height: 640
+      width: 640
```
Comment (alexrey88) [inline shared/trainer/defaults.yaml:39]: We should put maybe 560 (aka the same ratio than 224/256) for the crop size even tho we're ignoring it right now?

Comment (51N84D) [reply shared/trainer/defaults.yaml:39]: Why not

File: shared/trainer/defaults.yaml [hunk 2]
```diff
@@ -130,12 +130,12 @@ dis:
     get_intermediate_features: false
   p:
     <<: *default-dis
+    num_D: 3
```
[the remark this answers is not in the document]
Comment (51N84D) [reply shared/trainer/defaults.yaml:133]: I think it makes sense for only the default for the painter to be 3, while everything else is 1. The painter is really the only thing that needs multi-scale discrimination

File: shared/trainer/defaults.yaml [hunk 3]
```diff
@@ -12,14 +12,14 @@ tasks: [d, s, m, p] # [p] [m, s, d]
...
     train:
-      r: train_r.json
-      s: train_s.json
+      r: train_r_full.json
+      s: train_s_full.json
       rf: train_rf.json
     val:
-      r: val_r.json
-      s: val_s.json
+      r: train_r_full.json
```
Comment (melisandeteng) [inline shared/trainer/defaults.yaml:21]: this should be val_r_full.json and val_s_full.json

File: shared/trainer/defaults.yaml [hunk 4]
```diff
@@ -12,14 +12,14 @@ tasks: [d, s, m, p] # [p] [m, s, d]
...
+      s: train_s_full.json
       rf: val_rf.json
 
   loaders:
```
Comment (melisandeteng) [inline shared/trainer/defaults.yaml:25]: we should change batch_size to 4

**Pull request merged**